symfony/messenger/4.3/config/packages/messenger.yaml

```diff
@@ -0,0 +1,13 @@
+framework:
+    messenger:
+        # Uncomment the following line and choose a transports for collect all messages failed and try it later
+        # failure_transport: ...
+
+        transports:
+            # Uncomment the following lines to enable one of the transports named "amqp" or "sync"
```

Comment:
Instead of the comment, I think we should just link to `https://symfony.com/doc/current/messenger.html#transports`